src/components/CurrentWeather/CurrentWeather.js

```diff
@@ -3,15 +3,14 @@ import './CurrentWeather.css';
 import PropTypes from 'prop-types';
 
 export default function CurrentWeather({parks}){
+    console.log("PARKS:", typeof parks[0].latitude)
     const [weather, setWeather] = useState('')
     useEffect(() => {
         fetch(`https://api.openweathermap.org/data/2.5/weather?lat=${parks[0].latitude}&lon=${parks[0].longitude}&appid=${process.env.REACT_APP_API_SECRET_KEY}&units=imperial`)
         .then(res => res.json())
         .then(data => setWeather(data.main))
     }, [])
 
-    console.log(weather)
-
     return (
         <div>
             <img className='weather-img' src="https://cdn-icons-png.flaticon.com/256/1248/1248023.png"/>
```

Comment:
Since it might just a decorative element/icon, it may not need an alt, but I don't think it'd hurt!